services/tenant-ui/config/custom-environment-variables.json

```diff
@@ -5,6 +5,7 @@
     "showInnkeeperAdminLogin": "FRONTEND_INNKEEPER_SHOW_ADMIN",
     "showWritableComponents": "FRONTEND_TENANT_SHOW_WRITABLE_COMPONENTS",
     "showOIDCReservationLogin": "FRONTEND_SHOW_OIDC_RESERVATION_LOGIN",
+    "customReservationForm": "FRONTEND_TENANT_CUSTOM_RESERVATION_FORM",
```

Comment:
This isn't used anywhere?

Reply:
Yup. In the configmap.yaml.

Reply:
See comment there. I don't actually see `customReservationForm` used anywhere in the app. It's a file, not an env var isn't it?

Reply:
Hmmmm.... 
It's actually a combination of a file and Helm config.
I'll have a look. :eyes:

Reply:
Nice catch @loneil .